cfme/tests/automate/test_method.py

```diff
@@ -797,3 +845,88 @@ def test_delete_tag_from_category(custom_instance):
+    """
+    Polarion:
+        assignee: dgaikwad
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
+        setup:
+            1. Create domain, namespace, class and instance
+            2. Also create automate method with below ruby code:
+                >> $evm.log(:info, "Hello World")
+        testSteps:
+            1. Go to Configuration > Settings > Zones > Schedules
+            2. Create schedule with required fields:
+               >> Action - Automation Tasks
+               >> Object Details(Request) - Call_Instance
+               >> Attribute/Value Pairs
+                     >> domain - domain_name
+                     >> namespace - namespace_name
+                     >> class - class_name
+                     >> instance - instance_name
+               >> Timer Options
+            3. Check automation logs more than 1 times
+        expectedResults:
+            1.
+            2.
+            3. Automate method should be executed on scheduled time.
+
+    Bugzilla:
+        1713072
+    """
+    domain, namespace, auto_class, instance = custume_automate_setup
+
+    current_time, tz_num = current_server_time
+    start_date = current_time + relativedelta.relativedelta(minutes=5)
+    view = navigate_to(appliance.collections.system_schedules, 'Add')
+    available_list = view.form.time_zone.all_options
+
+    for tz in available_list:
+        if '{}:00'.format(tz_num[0:3]) in tz.text:
+            tz_select = tz.text
+            break
+
+    if round_min(start_date.minute) == 0:
+        start_date = start_date + relativedelta.relativedelta(minutes=60 - start_date.minute)
+        start_date_minute = str(start_date.minute)
+    else:
+        start_date_minute = str(round_min(start_date.minute))
+
+        attribute_value_pairs = {
+            "domain": domain.name,
+            "namespace": namespace.name,
+            "class": auto_class.name,
+            "instance": instance.name,
+        }
+
+    schedule = appliance.collections.system_schedules.create(
+        name=fauxfactory.gen_alphanumeric(),
+        description=fauxfactory.gen_alphanumeric(),
+        action_type="Automation Tasks",
+        request="Call_Instance",
+        attribute_value_pairs=attribute_value_pairs,
+        run_type="Hourly",
+        time_zone=tz_select,
+        start_hour=str(start_date.hour),
+        start_minute=start_date_minute,
+    )
+
+    @request.addfinalizer
+    def _finalize():
+        if schedule.exists:
+            schedule.delete()
+
+    matched_pattern = ".*INFO.* : Q-task_id.* Hello World"
+    for _ in range(2):
+        log = LogValidator("/var/www/miq/vmdb/log/automation.log",
+                           matched_patterns=[matched_pattern]
+                           )
+        log.start_monitoring()
+        wait = "15m" if _ == 0 else "75m"
+        log.validate(wait=wait)
```

Comment:
I think this will not work. From what I understand, it will validate the same statement twice. You can consider clearing logs before the second assertion.

Reply:
@digitronik need your opinion here.

Reply:
what I understand from testSteps, automation method should run on scheduled time.
Instantiating LogValidator two time so it will work but really I don't want test run for 1hr+.

@dgaikwad is it not possible to reduce this time? "75m" its too much :dagger:

Reply:
There are five run-types `['Once', 'Hourly', 'Daily',  'Weekly', 'Monthly']`
 and to run scheduler twice we have to select at least **Hourly** run type.
That's why I added chooses Hourly and required 1hr+ time
@digitronik  Please help me to reduce time is possible.

Reply:
@digitronik added code to reduce test time